README.md

```diff
@@ -1,3 +1,6 @@
+
+
+=======
```

Comment:
Can you remove all these 3 lines.

Reply:
@divyanshub024 Hey! as per your instruction I removed them.